contrib/python-bindings/source/triangulation_wrapper.cc

```diff
@@ -514,23 +514,24 @@ namespace python
 
     template <int dim, int spacedim>
     void
-    merge_triangulations(TriangulationWrapper &triangulation_1,
-                         TriangulationWrapper &triangulation_2,
-                         void *                triangulation)
+    merge_triangulations(boost::python::list &triangulations,
+                         void *               triangulation)
     {
       Triangulation<dim, spacedim> *tria =
         static_cast<Triangulation<dim, spacedim> *>(triangulation);
       tria->clear();
-      Triangulation<dim, spacedim> *tria_1 =
-        static_cast<Triangulation<dim, spacedim> *>(
-          triangulation_1.get_triangulation());
-      Triangulation<dim, spacedim> *tria_2 =
-        static_cast<Triangulation<dim, spacedim> *>(
-          triangulation_2.get_triangulation());
-      GridGenerator::merge_triangulations(*tria_1, *tria_2, *tria);
-      // We need to reassign tria to triangulation because tria was cleared
-      // inside merge_triangulations.
-      triangulation = tria;
```

Comment:
So this part is not necessary anymore?

Reply:
This is what happens inside

https://github.com/dealii/dealii/blob/e9461deb2e453f353d6bcab7d571aa2d4715ca47/source/grid/grid_generator.cc#L5976

Gigen that we pass the argument by reference, I don't see a need for this. Am I missing something?